bin/compile

```diff
@@ -39,6 +39,7 @@ case "${VERSION}" in
   5) VERSION="5.0.14";;
   6|6.2) VERSION="6.2.12";;
   7|7.0) VERSION="7.0.11";;
+  7.2|7.2.5) VERSION="7.2.5";;
```

Comment:
The purpose of this `case` statement is to add support for version aliases - ie: specifying a partial version and have it map to the latest patch release of that version. As such, only partial versions need to be included here - the exact versions themselves don't need to be mapped.

ie this should be:

```bash
  7.2) VERSION="7.2.5";;
```

And then later, when you want to make it the default (for both the empty string and the "7" case), it would become:

```bash
  7.0) VERSION="7.0.11";;
  7|7.2) VERSION="7.2.5";;
```